### PR TITLE
Add support for 0.12.0

### PR DESCRIPTION
**Pull request by kaxil**

### CODE REVIEW

File: src/astronomer-platform [hunk 1]
```diff
@@ -6,6 +6,12 @@
       "level": "bug_fix",
       "url": "https://github.com/astronomer/astronomer/releases/tag/v0.11.1",
       "release_date": "2020-01-31T00:00:00+00:00"
+    },
+    {
+      "version": "0.12.0",
+      "level": "bug_fix",
```
Comment (schnie) [inline src/astronomer-platform]: @kaxil what would a good name for new features be here? This was more than bug fixes.

Comment (kaxil) [reply src/astronomer-platform]: How about "minor_release" ?

Comment (schnie) [reply src/astronomer-platform]: That might work. @ashb did you have any different naming conventions in mind when you came up with the spec?

Comment (andriisoldatenko) [reply src/astronomer-platform]: sounds like minor_release :)

Comment (schnie) [inline src/astronomer-platform]:
```suggestion
      "level": "minor_release",
```

**Pull request merged**